### ISSUE TITLE
What causes the anti-collision?

### ISSUE THREAD

**Issue by RanvirChoudhary** [johnBuffer/NoCol#13]
Does anyone know what causes the anti-collision?

**Comment by cluder** [comment]
yes it would be cool if you could explain it a bit more in the readme what bug happens

**Comment by KosmosisDire** [comment]
I believe this is simply a statistical phenomenon. I believe it's a simple circle based collision system, basically without any friction or loss of energy. When they collide with another particle it pushes them out of the way and over time it stabilizes into a system without any collisions. This is only what it looks like to me, I could be wrong.

**Comment by nezvers** [comment]
The orbiting happens against center of the screen. They are colliding at the beginning and push each other out of the way until they form stable oscillation over time. They touch each other for a while and setting in non colliding oscillation.
